CSE/Logic/DataManagement/Converter.cs

```diff
@@ -5,20 +5,27 @@
 using System.Linq;
 using System.IO;
 using System;
+using System.Threading;
 using Logic.Database;
 using System.Diagnostics;
 
 namespace Logic.DataManagement
 {
-    public class Converter : IConverter
+	public class Converter : IConverter
     {
-        private readonly ITextProcessing _textProcessing;
-        private readonly IUpdater _updater;
```

Comment:
Why have you deleted IUpdater field?

Reply:
Accident made when resolving conflicts, my bad